jetty/src/test/scala/org/http4s/server/jetty/Issue454.scala

```diff
@@ -44,6 +44,9 @@ object Issue454 {
     service      = HttpService {
       case req@GET -> Root => Ok(insanelyHugeData)
     },
-    servletIo    = org.http4s.servlet.NonBlockingServletIo(4096)
+    servletIo    = org.http4s.servlet.NonBlockingServletIo(4096),
+    threadPool   = org.http4s.util.threads.newDefaultFixedThreadPool(
+      4, org.http4s.util.threads.threadFactory(i => s"org.http4s.server.jetty.DefaultExecutor-$i")
+    )
```

Comment:
I added a thread pool to `Http4sSpec`'s companion object for these "I just need a thread pool, any old one will do" cases.